app/assets/stylesheets/welcome.scss

```diff
@@ -27,27 +68,69 @@ $backdrop : #C9D6AE;
    width: 100%;
 }
 
+.inherit {
+  width: inherit;
+  margin-top: 30px;
+  padding: 6px;
+  input {
+    width: 100%;
+    cursor: pointer;
+    color: white;
+  }
+}
+
+input[type='checkbox'] {
+  width: 100%;
+  cursor: pointer;
+}
+
+input[type='checkbox'] {
+    -webkit-appearance:none;
+    width:30px;
+    height:30px;
+    background:white;
+    border:2px solid gray;
+    margin: 0;
+    margin-right: 10px
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should be terminated by a semicolon